PWA/src/components/Timesheet/ConfirmSubmission.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <v-row justify="center">
+    <v-btn color="success" dark @click.stop="dialog = true" @click="validate">
+      Submit
+    </v-btn>
+
+    <v-dialog v-model="dialog" max-width="300">
+      <div v-if="valid">
+        <v-card>
+          <div v-if="!loading">
+            <v-card-title class="headline" id="confirm"
+              >Are you sure want to submit the form?</v-card-title
+            >
+
+            <v-card-text>
+              Some text talking about how this submission is final unless
+              something is wrong with it.
+            </v-card-text>
+
+            <v-card-actions>
+              <v-spacer></v-spacer>
+
+							<!-- Confirm if user is ready to submit -->
+              <v-btn color="red" text @click="dialog = false">
```

Comment:
Since this has more than 2 props, I think each prop should be put into its own line